### PR TITLE
fix for ovs-4456

### PR DESCRIPTION
**Pull request by domsj**
proxy startup takes too long when removing a large fragment cache

### CODE REVIEW

File: ocaml/src/fragment_cache.ml [hunk 1]
```diff
@@ -79,24 +79,36 @@ let read_it fd ~len =
...
+  | true ->
+      let cmd = Printf.sprintf "mv %s %s-old-%f" dir dir (Unix.gettimeofday ()) in
       Lwt_log.debug_f "cmd:%S" cmd >>= fun () ->
       let command = Lwt_process.shell cmd in
       Lwt_process.exec command >>= fun status ->
       match status with
-      | Unix.WEXITED 0 -> Lwt_log.debug_f "cmd:%S ok" cmd
+      | Unix.WEXITED 0 ->
+         Lwt_log.debug_f "cmd:%S ok, now removing in the background" cmd >>= fun () ->
+         Lwt_process.exec (Printf.sprintf "mkdir %s" dir |> Lwt_process.shell) >>= fun _ ->
```
Comment (toolslive) [inline ocaml/src/fragment_cache.ml]: don't we want to check this  process status?

Comment (domsj) [inline ocaml/src/fragment_cache.ml]: My laziness didn't want to check it at first, but you're right, we should, and I will in another commit.

Comment (domsj) [inline ocaml/src/fragment_cache.ml]: commit added

**Comment by toolslive** [comment]
LGTM

**Pull request merged**